### DISCUSSION TITLE
widget setting for model

### DISCUSSION THREAD

**Discussion by bobcolo** [EdgeTX/edgetx#5309] [Ideas]
hello 
i think it should be very useful if widget setting could be done on PC with companion 
it can be done on simulation process but it's not possible to save 
can you think it will be possible in the futur

thanks

**Comment by pfeerick** [comment]
If you run the simulator in standalone mode, it is possible for settings
changes to be persistent. However, built in widgets, it may be possible (to
configure in Companion in the future), but for Lua widgets it may be
somewhat difficult to do it reliably. As that would require you to  always
ensure the copy of the radio SD that you keep on your computer always
matches that of the radio... Or more specifically, that the versions of the
Lua widgets are the same, so that the optiond the widget has can be
queried.

**Reply by bobcolo** [reply]
thanks 
i will try standalone mode if i found it 
as i dont have sensitive screen it,s more easy on PC

**Reply by bobcolo** [reply]
it was the good solution
that works perfectly
thanks
